mpesa_test.go

```diff
@@ -650,3 +651,116 @@ func TestMpesa_STKPushQuery(t *testing.T) {
 		})
 	}
 }
+
+func Test_RegisterC2BURL(t *testing.T) {
+	asserts := assert.New(t)
+	tests := []struct {
+		name       string
+		env        Environment
+		mock       func(t *testing.T, ctx context.Context, app *Mpesa, c *mockHttpClient, c2bRequest RegisterC2BURLRequest)
+		c2bRequest RegisterC2BURLRequest
+	}{
+		{
+			name: "it should register URLs in sanbox",
+			env:  Sandbox,
+			c2bRequest: RegisterC2BURLRequest{
+				ShortCode:       600638,
+				ResponseType:    "Complete",
```

Comment:
@weezqyd the tests are failing here
```suggestion
				ResponseType:    "Completed",
```